isis/scripts/isisStartup.csh

```diff
@@ -0,0 +1,37 @@
+#!/bin/csh
+################################################################################
+# This file should be sourced within your current shell using the "source"
+# command.
+#
+# On the command line type:
+# > setenv ISISROOT ????
+# > source isis3Startup.csh
+#
+# Replace the "????" in the above command line with the path you installed
```

Comment:
Should we merge https://github.com/USGS-Astrogeology/ISIS3/pull/3927 before merging this?

Reply:
Ugg, somehow I forgot to change branches.